app/common/renderer/polyfills.js

```diff
@@ -1,39 +1,18 @@
-let settings, clipboard, shell, ipcRenderer, i18NextBackend, i18NextBackendOptions, fs, util;
+/**
+ * The '#local-polyfills' alias is defined in both Vite config files.
+ * Since both files define different resolution paths,
+ * they cannot be added to tsconfig and eslint configurations
+ */
 
-function buildForBrowser() {
-  if (process.env.BUILD_BROWSER) {
-    return true;
-  }
-
-  if (typeof navigator !== 'undefined' && !/electron/i.test(navigator.userAgent)) {
-    return true;
-  }
-
-  return false;
-}
-
-if (buildForBrowser()) {
-  ({
-    settings,
-    clipboard,
-    shell,
-    ipcRenderer,
-    i18NextBackend,
-    i18NextBackendOptions,
-    fs,
-    util,
-  } = require('../../web/polyfills'));
-} else {
-  ({
-    settings,
-    clipboard,
-    shell,
-    ipcRenderer,
-    i18NextBackend,
-    i18NextBackendOptions,
-    fs,
-    util,
-  } = require('../../electron/renderer/polyfills'));
-}
+import {
```

Comment:
True, but I think it's cleaner to do this, rather than adding `eslint-disable-line` in many more files. Plus since neither VS Code nor ESLint are aware of this alias, the imports won't be auto-adjusted if file locations change, so having it all in one place would also reduce the manual effort.

Reply:
why `eslint-disable-line ` would be needed? Are we talking about the same thing: 

```js
export {
  settings,
  clipboard,
  shell,
  ipcRenderer,
  i18NextBackend,
  i18NextBackendOptions,
  fs,
  util,
} from '#local-polyfills'
```

Reply:
Yes - this last line currently has the `eslint-disable-line` comment, and if I remove the comment, I get `Unable to resolve path to module '#local-polyfills'`. Normally this would be resolved by adding the alias to the ESLint config file, but since the alias can point to two different locations, I thought it would better to just disable ESLint and leave an explanatory comment above it.

Reply:
makes sense. Make leave a todo there for now

Reply:
Okay, I think I misunderstood, I didn't realise the two lines can just be combined. Will adjust this.